includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -81,13 +85,52 @@ public function get_selector_conversion_mapping() {
 		];
 	}
 
+	/**
+	 * Convert picture element into image element or mark as px verified.
+	 *
+	 * @return void
+	 */
+	protected function process_picture_elements() {
+
+		$picture_img_query = $this->dom->xpath->query( '//picture/img' );
+		if ( 0 === $picture_img_query->length ) {
+			return;
+		}
```

Comment:
I guess this is redundant. The `$this->dom->xpath->query()` [returns](https://www.php.net/manual/en/domxpath.query.php) a `DOMNodeList`. If it's an empty list, the `foreach` loop below will do 0 iterations.